Modules/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -1657,10 +1661,27 @@ private function addCensorshipObject()
         $this->handleCensorship();
     }
 
+    /**
+     * @return string
+     */
+    private function getModalActions()
+    {
+        $modalString = '';
+        foreach ((array) $this->modalActionsContainer as $modal) {
+            global $DIC;
+
+            $modalString .= $DIC->ui()->renderer()->render($modal);
+        }
+        return $modalString;
+    }
+
     private function handleCensorship($wasRevoked = false)
     {
         if (!$this->objCurrentTopic->isClosed() && $this->is_moderator) {
             $message = $this->handleFormInput($_POST['formData']['cens_message']);
+            if($message === ''){
```

Comment:
Please check our/respect the PSR-2 code styles by applying the `php-cs-fixer` to all modified files.